src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/TelemetryReportingLSPRequestInvoker.cs

```diff
@@ -78,12 +80,16 @@ public TelemetryReportingLSPRequestInvoker(LSPRequestInvoker requestInvoker, ITe
         }
     }
 
-    private IDisposable? Track(string name, string method, string languageServerName)
+    internal IDisposable? Track(string name, string method, string languageServerName, Guid correlationId = default)
```

Comment:
I assume `default` is the same as `Guid.Empty`, but might be better to use explicitly?

Reply:
You can't use `Guid.Empty` as a parameter default because it isn't a constant. Same as the reason you can't use `string.Empty` but you can use `""`.